azure/src/test/java/com/microsoft/azure/AzureTests.java

```diff
@@ -76,37 +72,6 @@ public void setup() throws Exception {
     }
 
     /**
-     * Tests the public IP address implementation from an individual service client
-     * @throws Exception 
-     */
-    @Test
-    public void testPublicIpAddressesInGroup() throws Exception {
```

Comment:
I see. Didn't see that.
Can we move them into seperate packages?
No particular reason, just feel like the roll-up client should just be a wrapper on top of tested robust individual clients running underneath.
When a test fails we also want to catch it earlier in building the individual client packages instead of finding it in a later phase like the roll-up package.